generatebundlefile/ecr_test.go

```diff
@@ -90,17 +90,15 @@ func TestSplitECRName(t *testing.T) {
 		t.Run(tc.testName, func(tt *testing.T) {
 			got, got2, err := splitECRName(tc.testHelmName)
 			if (err != nil) != tc.wantErr {
-				t.Fatalf("splitECRName() error = %v, wantErr %v", err, tc.wantErr)
+				tt.Fatalf("splitECRName() error = %v, wantErr %v", err, tc.wantErr)
```

Comment:
You probably don't want `Fatal` here. That'll stop the test prematurely.
I think all you really need is this:
```go
tc.wantErr && !assert.Error(tt, err) || assert.NoError(tt, err)
```
If the compiler complains about that, go long-form:
```go
if tc.wantErr {
    assert.Error(tt, err)
} else {
    assert.NoError(tt, err)
}
```
There are lots of ways that I think would be a bit cleaner... you could replace wantErr instead with either assert.Error or assert.NoError for example... But all this can be done later.